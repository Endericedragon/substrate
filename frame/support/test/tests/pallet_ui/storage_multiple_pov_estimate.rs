#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::*;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(_);

	#[pallet::error]
	pub enum Error<T> {}

	#[pallet::storage]
	#[pallet::pov_estimate = MaxEncodedLen]
	#[pallet::pov_estimate = MaxEncodedLen]
	type Foo<T: Config> = StorageValue<_, u8>;
}

fn main() {
}
