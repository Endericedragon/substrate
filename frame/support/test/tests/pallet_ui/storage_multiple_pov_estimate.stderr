error: Invalid attribute: Duplicate attribute
  --> tests/pallet_ui/storage_multiple_pov_estimate.rs:16:3
   |
16 |     #[pallet::pov_estimate = MaxEncodedLen]
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
